packages/sequence/src/index.ts

```diff
@@ -10,53 +10,54 @@ function sequence(options?: SequenceOptions): WalletInit {
 
   return () => {
     // @ts-ignore
-    return window?.ethereum && window.ethereum?.isSequence ?
-      []
-    :
-     {
-      label: 'Sequence',
-      getIcon: async () => (await import('./icon.js')).default,
-      getInterface: async () => {
-        const { sequence } = await import('0xsequence')
-        const { createEIP1193Provider } = await import('@web3-onboard/common')
-
-        const instance = new sequence.Wallet(network);
-
-        if (!instance.isConnected()) {
-          const connectDetails = await instance.connect({
-            app: appName,
-            authorize: true
-          });
-
-          if (!connectDetails.connected) {
-            throw new Error('Failed to connect to the wallet')
-          }
-        }
+    return window?.ethereum && window.ethereum?.isSequence
```

Comment:
You can revert the formatting within this file so you dont get tagged in the gitblame should any errors arise 😄

Reply:
uh oh... Done. There are also changes to the wallet connect package. Should I revert that as well?

Reply:
Great - reverted the changes. Always happy to have the issues not point to me lol(hard task). My pleasure!

Reply:
Excellent, I made a handful of changes if you want to pull. Ill run some final checks and testing and we should be ready to rock 🎸